packages/xpetra/src/CrsMatrix/Xpetra_CrsMatrix.hpp

```diff
@@ -260,6 +260,10 @@ namespace Xpetra {
      */
     virtual void apply(const MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node > &X, MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node > &Y, Teuchos::ETransp mode=Teuchos::NO_TRANS, Scalar alpha=ScalarTraits< Scalar >::one(), Scalar beta=ScalarTraits< Scalar >::zero()) const = 0;
 
+    //! matrix-multivector multiplication for region layout matrices
+    virtual void apply(const MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node > &X, MultiVector< Scalar, LocalOrdinal, GlobalOrdinal, Node > &Y, Teuchos::ETransp mode, Scalar alpha, Scalar beta, bool sumInterfaceValues, const RCP<Import<LocalOrdinal, GlobalOrdinal, Node> >& regionInterfaceImporter, const Teuchos::ArrayRCP<LocalOrdinal>& regionInterfaceLIDs) const = 0;
```

Comment:
@pohm01 Please make sure to document `regionInterfaceImporter` and `regionInterfaceLIDs`. We need some form of description, such that a user knows, what he/she has to provide.